uninstall.php

```diff
@@ -5,43 +5,50 @@
  * @package Jetpack
  */
 
-use Automattic\Jetpack\Sync\Sender;
 use Automattic\Jetpack\Backup\Helper_Script_Manager;
+use Automattic\Jetpack\Sync\Sender;
 
-if (
-	!defined( 'WP_UNINSTALL_PLUGIN' )
-	||
-	!WP_UNINSTALL_PLUGIN
-	||
-	dirname( WP_UNINSTALL_PLUGIN ) != dirname( plugin_basename( __FILE__ ) )
-) {
-	status_header( 404 );
-	exit;
-}
-
-if ( ! defined( 'JETPACK__PLUGIN_DIR' ) ) {
-	define( 'JETPACK__PLUGIN_DIR', plugin_dir_path( __FILE__ )  );
+/**
+ * Uninstall script for Jetpack.
+ */
+function jetpack_uninstall() {
+	if (
+		! defined( 'WP_UNINSTALL_PLUGIN' )
+		||
+		! WP_UNINSTALL_PLUGIN
+		||
+		dirname( WP_UNINSTALL_PLUGIN ) !== dirname( plugin_basename( __FILE__ ) )
```

Comment:
Since you're touching this anyway, feel like cleaning this up? I don't care much whether you put the `||` at the start or end of the lines, but this is just weird.

Reply:
yessir 433e64cc2